tests/tests_core/test_results.py

```diff
@@ -37,6 +38,34 @@ def load_example_results():
 
 	return r.from_hdf(example_fpath)
 
+def mock_results_object(data, kpi_subset=None, derived_kpis=None, **kwargs):
+	"""
+	Create a results object for any tests involving fixed_horizon_delta()
```

Comment:
why factor out this functionality as a separate function? is it going to be used anywhere besides `fixed_horizon_delta()`?

addendum: would this code persist in Experiment.delta() further?